wa/commands/create.py

```
@@ -35,6 +45,196 @@
+            return
+        # Create the database (override if exists)
+        if args.dbname == 'postgres':
+            raise ValueError('Databasename to create cannot be postgres.')
+
+        possible_error_list = [
+            (
+                re.compile('FATAL:  role ".*" does not exist'),
+                'Username does not exist or password is incorrect'
+            ),
+            (
+                re.compile('FATAL:  password authentication failed for user'),
+                'Password was incorrect'
+            ),
+            (
+                re.compile('fe_sendauth: no password supplied'),
+                'Passwordless connection is not enabled. '
+                'Please enable trust in pg_hba for this host '
+                'or use a password'
+            ),
+            (
+                re.compile('FATAL:  no pg_hba.conf entry for'),
+                'Host is not allowed to connect to the specified database '
+                'using this user according to pg_hba.conf. Please change the '
+                'rules in pg_hba or your connection method'
+            ),
+            (
+                re.compile('FATAL:  pg_hba.conf rejects connection'),
+                'Connection was rejected by pg_hba.conf'
+            ),
+        ]
+
+        def predicate(error, handle):
+            if handle[0].match(str(error)):
+                raise CommandError(handle[1] + ': \n' + str(error))
+
+        try:
+            self.create_database(args)
+        except OperationalError as e:
+            for handle in possible_error_list:
+                predicate(e, handle)
+            raise e
+
+        # Update config file
+        with open(args.config_file, 'r') as config_file:
+            config = yaml.load(config_file)
+        if 'postgres' in config:
+            if not args.force_update_config:
+                self.logger.warning("The entry 'postgres' already exists in the config file. " +
```

Raise `CommandError` here instead of logging a warning.